package.json

```diff
@@ -59,6 +59,7 @@
     "express": "^4.18.1",
     "express-http-proxy": "^1.6.3",
     "filesize": "^10.0.5",
+    "get-graphql-schema": "^2.1.2",
```

Comment:
This is not needed as it will be installed globally with `npx -y get-graphql-schema`

```suggestion
```